src/pcompass/pcompass/static/stylesheets/scss/general/_variables_theme.scss

```diff
@@ -20,4 +20,3 @@ $color-brand-two-extrovert: $color-brand-one-extrovert;
 $color-text-normal: #4d4d4d;
 $color-text-extrovert: #1c1c1b;
 $color-text-highlight-normal: $color-brand-one-normal;
-
```

Comment:
@xi said pcompass is an exception: we want to have the same text color as the surrounding site.